sciLifeLab_utils/run_delivery.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import os
+import glob
+import re
+import subprocess
+
+def main(arg):
+    project = os.path.split(os.path.realpath(arg.source))[1]
+    move_from_path = "{}/*/results/".format(arg.source)
+    pathway = glob.glob(move_from_path)
+    pattern = re.compile(r"/[^/]+/[^/]+/[^/]+/[^/]+/[^/]+/([^/]+)/") #Matches the group betwen the 6th / /
```

Comment:
Ah, if you have the trailing slash then it will give `results`, yeah. You need a second `dirname` then:
```python
os.path.basename( os.path.dirname( os.path.dirname( path ) ) )
```
..getting ugly..